queries/wit/highlights.scm

```diff
@@ -0,0 +1,84 @@
+; Comments
+(line_comment) @comment
+
+(block_comment) @comment
+
+; Primitive Types
+[
+  "bool"
+  "s8"
+  "s16"
+  "s32"
+  "s64"
+  "u8"
+  "u16"
+  "u32"
+  "u64"
+  "float32"
+  "float64"
+  "char"
+  "string"
+] @type.builtin
+
+; Container Types
+[
+  "list"
+  "tuple"
+  "option"
+  "result"
+] @type.builtin
+
+"func" @keyword.function
+
+; Keywords for file structure and components
+[
+  "record"
+  "enum"
```

Comment:
we do have `@keyword.type` for things like enum, so you can probably use it?

Reply:
Nice one, thanks! Done ✅